views/newsletter-page.js

```diff
@@ -0,0 +1,83 @@
+
+$(document).ready(function(){
+	createNewsletterTemplate ();
+	enableSubmitClick ();
+})
+
+function createNewsletterTemplate () {
+	let template = $('#newsletter-page-template').html();
+	let compiledTemplate = Handlebars.compile(template);
+	$('#application').html(compiledTemplate());
+}
+
+function enableSubmitClick () {
+    let forms = document.getElementsByClassName('needs-validation');
+    let validation = Array.prototype.filter.call(forms, function(form) {
+    	let button = document.getElementById('button');
+		button.onclick = function () {
+			form.addEventListener('submit', function(event) {
+	        if (form.checkValidity() === false) {
+	          event.preventDefault();
+	          event.stopPropagation();
+	          console.log('sprobuj jeszcze raz')
+	        }
+	        form.classList.add('was-validated');
+	        event.preventDefault();
+	        savePersonalData ();
+	        form.innerHTML = '';
+	        document.getElementById('welcome-text').innerHTML = '';
+	        document.getElementById('thank-you-text').classList.remove('d-none')
+	        console.log('hurra')
+
+      	}, false);
+	 	}
+	});
+}
+
+function createThankYouTemplate () {
+	let template = $('#thank-you-template').html();
+	let compiledTemplate = Handlebars.compile(template);
+	$()
+}
+
+function savePersonalData () {
+	var forms = document.getElementsByClassName('needs-validation');
+	Array.from(forms).forEach(function(form){
+		if (form.classList.contains('was-validated') == true) {
+			console.log('grzybek')
+			const emailInput = document.getElementById('email-input');
```

Comment:
Since you're using jQuery already, you can use its simpler notation: 
`const emailInput = $('#email-input');`

Note that it doesn't return a DOM element, but a jQuery object associated with this DOM element, so you won't be able to read the `value` property immediately. But thankfully, jQuery provides a `val` method on its objects.
So you can go straight to:
`const email = $('#email-input').val();`